WBooks/ios/WBooks.xcodeproj/project.pbxproj

```diff
@@ -40,6 +39,8 @@
 		ADBDB9381DFEBF1600ED6528 /* libRCTBlob.a in Frameworks */ = {isa = PBXBuildFile; fileRef = ADBDB9271DFEBF0700ED6528 /* libRCTBlob.a */; };
 		ED297163215061F000B7C4FE /* JavaScriptCore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = ED297162215061F000B7C4FE /* JavaScriptCore.framework */; };
 		ED2971652150620600B7C4FE /* JavaScriptCore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = ED2971642150620600B7C4FE /* JavaScriptCore.framework */; };
+		6823AF0B11E14638B26FA724 /* libRNGestureHandler.a in Frameworks */ = {isa = PBXBuildFile; fileRef = A07E6A5D4DC2428BA899F2AB /* libRNGestureHandler.a */; };
```

Comment:
Remove all files from the repository

Reply:
what files specifically? There are a lot of folders and i don't know which can i delete and which don't

Reply:
The vscode folder i suppose but in the folder android or ios i don't know.